.github/workflows/main.yml

```diff
@@ -5,6 +5,7 @@ on:
     tags-ignore: [dev]
   pull_request:
     branches: [main]
+  workflow_dispatch:
```

Comment:
I added this to be able to test the CI easier but I don't see a reason not to have this trigger for this workflow.

Reply:
Sure!